sources/tdigest.c

```diff
@@ -0,0 +1,322 @@
+#include <stdlib.h>
+#include <stdbool.h>
+#include <string.h>
+#include <math.h>
+
+#include "tdigest.h"
+
+#define M_PI 3.14159265358979323846
```

Comment:
what's wrong with math.h's Pi? Not enough digits? Don't we need double suffix here?